src/main/java/org/parabot/core/Context.java

```diff
@@ -163,7 +165,9 @@ public Applet getApplet() {
     /**
      * Loads the game
      */
-    public void load() {
+
+    public void load() throws FailToParseHooksException, FieldNotFoundException {
+        BotUI.getInstance().getJMenuBar().remove(2);
```

Comment:
I don't know. Must have sneaked in when I was merging development. I guess I could remove it..